scavenger-agent-python/README.md

```diff
@@ -96,4 +97,4 @@ app = Flask(__name__)
 def hello_world():
     return 'Hello World!'
 ```
-* 코드를 직접 읽어서 메서드를 탐색해야 하므로 pyc 파일은 현재 지원되지 않습니다.
+* Pyc files are currently not supported because Python code must be read by the agent.
```

Comment:
```suggestion
* The direct Instrumentation for pyc files are currently not supported because Python code must be read by the scavenger agent.
```

Reply:
In that case, how about updating Korean too?

Reply:
good!!